### DISCUSSION TITLE
firecheckout collect and show data in my account page

### DISCUSSION THREAD

**Discussion by wdmarketLV** [breezefront/community#93] [1. Help]
Hello, I am using fire checkout. 
I need to collect data in checkout and then show them on my account page. 

Like I can make custom checkout fields and also I can make customer fields. 
But on my account page, I will only see customer fields. 
So How Can I display customer?

**Comment by vovayatsyuk** [comment]
I'm working on swissup/customer-field-manager and swissup/address-field-manager integrations and will release their updates on this week.

**Comment by vovayatsyuk** [comment]
Please update swissup modules to their latest versions to get Breeze integrations:

```
composer update "swissup/*" -W
```
